frontend/src/store/ConfigStore.ts

```diff
@@ -68,7 +68,7 @@ const persistConfig = {
 const composeEnhancers =
   (process.env.NODE_ENV === 'development' && window && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose;
 
-const configureStore = (initialState: KialiAppState): Store<KialiAppState, KialiAppAction> => {
+export const configureStore = (initialState: KialiAppState): Store<KialiAppState, KialiAppAction> => {
```

Comment:
Does this need the export?

Reply:
Nope this was leftover from testing but I ended up not using this. I'll remove.